### DISCUSSION TITLE
I want to know how to use iconify-icon.

### DISCUSSION THREAD

**Discussion by LibraryLibrarian** [saicaca/fuwari#25] [Q&A] [answered]
In `src/config.ts` ProfileConfig, it was explained that iconify-icons other than fa6-brands can be used.
If you want to use icons other than the default iconify-icon (for example, `simple-icons:misskey`), how should you configure it?
Is there no other way than directly editing `Layout.astro`?

**Comment by saicaca** [comment, marked as answer]
You will need to install the corresponding icon set `@iconify-json/<icon-set-name>`

For example, run `pnpm add @iconify-json/simple-icons` to add `simple-icons`.

**Reply by LibraryLibrarian** [reply]
Thank you, it has been resolved without any problems.
It was a very simple issue.
I will study a bit more about Astro and pnpm.
